pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -571,9 +635,23 @@ def get(self, request, *args, **kwargs):
             if guard.TYPE == "content_redirect":
                 return redirect(guard.preauthenticate_url(url))
 
+        CollectionArtifactDownloadView.log_download(
+            request, self.kwargs["filename"], distro_base_path
+        )
+
         return redirect(url)
 
 
+class CollectionDownloadLogViewSet(viewsets.ReadOnlyModelViewSet):
+    """Collection download viewset."""
+
+    model = CollectionDownloadLog
+    serializer_class = CollectionDownloadLogSerializer
+    filterset_class = CollectionDownloadLogFilter
+    filter_backends = (DjangoFilterBackend,)
```

Comment:
I think, this is set by default in the drf-settings already. This line should not be necessary.